R/logging-tools.R

```diff
@@ -77,7 +77,6 @@ log_button <- function(data_storage, input, button_id) {
 
 #' @details Each function (except \code{log_custom_action}) store logs inside
 #' 'user_log' table. It is required to build admin panel
-#' (See \link{prepare_admin_panel_components}).
```

Comment:
I had to remove this because this function is no longer exported in the package NAMESPACE, so the link will be invalid and cause a fail in `R CMD CHECK`.